accapp.rb

```diff
@@ -29,6 +29,25 @@
 ## Project list & general endpoint functions ##
 ###############################################
 
+#  preview
+post '/preview' do
+  class Preview
+    def initialize (url, integration)
+      @url = url
+      @integration = integration
+    end
+    def url
+      @url
+    end
+    def integration
+      @integration
+    end
+  end
```

Comment:
Considering how little this class does I might use a struct instead. Either way, though.